react/features/polls/reducer.ts

```diff
@@ -83,21 +83,29 @@ ReducerRegistry.register<IPollsState>('features/polls', (state = INITIAL_STATE,
         // if the poll exists, we update it with the incoming answer
         const newAnswers = state.polls[pollId].answers
             .map(_answer => {
+                const answerVoters = _answer.voters
```

Comment:
Can you pl add a comment here that explains you are supporting the old data model?